src/Button.js

```diff
@@ -74,7 +85,13 @@ class AriaMenuButtonButton extends React.Component {
       onBlur: this.context.ambManager.handleBlur
     };
 
-    specialAssign(buttonProps, props, checkedProps);
+    const reserved = Object.assign({}, checkedProps);
+    // The disabled property should be passed down to the Button element
+    // if the tag has support for disabled attribute
+    if (disabledSupportedTags().find(r => props.tag === r)) {
```

Comment:
Let's use `indexOf` for browser support.

Reply:
Sure!